torchvision/models/quantization/mobilenetv3.py

```diff
@@ -16,16 +17,17 @@
 }
 
 
-class QuantizableSqueezeExcitation(SqueezeExcitation):
+class QuantizableSqueezeExcitation(SElayer):
     def __init__(self, *args: Any, **kwargs: Any) -> None:
+        kwargs["scale_activation"] = nn.Hardswish
         super().__init__(*args, **kwargs)
         self.skip_mul = nn.quantized.FloatFunctional()
```

Comment:
Should we be concerned about BC here? what if some downstream callsites construct the module with `squeeze_factor`?

Reply:
Good question. An option would be to keep the old class, remove its usages and deprecate it and introduced a new one similar to what I did here:
https://github.com/pytorch/vision/blob/496940877e3cb5164408a6a4b06a65364734d43b/torchvision/models/mobilenetv3.py#L22-L26

I had a quick look both on [Github](https://github.com/search?p=1&q=QuantizableSqueezeExcitation&type=Code) and on FBcode and I can't find any direct usages of this class outside the intended in this module. In addition the module is undocumented and not publicly exposed to `__all__`, hence I would personally be OK offering this level of BC. Thoughts?